src/particle/pdgid/__init__.py

```diff
@@ -61,6 +61,15 @@
 
 from __future__ import absolute_import
 
+from typing import Tuple
+
+from .functions import is_composite_quark_or_lepton  # noqa: F401
+from .functions import is_gauge_boson_or_higgs  # noqa: F401
+from .functions import is_generator_specific  # noqa: F401
+from .functions import is_quark  # noqa: F401
+from .functions import is_sm_gauge_boson_or_higgs  # noqa: F401
+from .functions import is_special_particle  # noqa: F401
+from .functions import is_technicolor  # noqa: F401
```

Comment:
Are all these really supposed to be missing from `__all__`? `is_composite_quark_or_lepton` I assume yes, but `is_quark`, for example?

Reply:
I fixed a bunch of these when adding several new PDGID related functions in https://github.com/scikit-hep/particle/pull/324. If you could rebase on master as soon as that is merged some will be gone for sure. I will then check carefully what exactly we want public ... Advance thanks.

Reply:
Sure, will do!

Reply:
You can now rebase and then I will have a review for what should be public or private ...

Reply:
Done.